LJAPI.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+import os
+import requests
+import transaction
+
+
+from LearningJournal.models import Entry
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from zope.sqlalchemy import ZopeTransactionExtension
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+
+
+database_url = os.environ.get('DATABASE_URL', None)
+engine = engine_from_config({'sqlalchemy.url': database_url})
+DBSession.configure(bind=engine)
+
+json_entries = {}
+
+
+def get_json():
+    """Get and store journal json data."""
+    resp = requests.get('https://sea401d2.crisewing.com/api/export?apikey=' + os.environ.get('API_KEY'))
+
+    if resp.status_code == 200:
+        json_entries = resp.json()
+        return json_entries
+    else:
+        return None
+
+
+def search_and_create(new_title, new_text):
+    """See if entries already exist, then add to DB."""
+    try:
+        in_db = DBSession.query(Entry).filter(Entry.title == new_title)
+        DBSession.query(Entry).filter(Entry.id == in_db.id).update({'title': new_title, "text": new_title})
```

Comment:
This looks broken to me.  If I understand your code correctly, it appears that `in_db` will be a query object.  That object may or may not contain an Entry instance.  But you don't know as you haven't checked it.  Then you are querying again to get the entry that has the same id as the entry you just got by filtering on title.  Why do that?  Why not just use the one Entry you already have, which is either the right one, or isn't one at all, and update that entry?  There's no reason to do the same work twice in a row in two different ways.